app/models/study_file.rb

```diff
@@ -25,6 +25,14 @@ class StudyFile
   STUDY_FILE_TYPES = ['Cluster', 'Coordinate Labels' ,'Expression Matrix', 'MM Coordinate Matrix', '10X Genes File',
```

Comment:
I'm wondering if maybe this might be simpler/cleaner.  I could take it or leave it.  Either way, we should `freeze` all these constants.
```suggestion
# hash of file_type to display names
STUDY_FILE_TYPE_NAME_HASH = {
  'Cluster' => 'Cluster', 
  'Coordinate Labels' => 'Coordinate Labels',
  'Expression Matrix', => 'Dense matrix',
  'MM Coordinate Matrix' => => 'Sparse matrix (.mtx)', 
  '10X Genes File' => '10X Features File',
  '10X Barcodes File' => '10X Barcodes File', 
  'Gene List' => 'Gene List', 
  'Metadata' => 'Metadata', 
  'Fastq' => 'Fastq', 
  'BAM' => 'BAM', 
  'BAM Index' => 'BAM Index', 
  'Documentation' => 'Documentation',
  'Other' => 'Other', 
  'Analysis Output' => 'Analysis Output', 
  'Ideogram Annotations' => 'Ideogram Annotations', 
  'Image' => 'Image'
}.freeze

STUDY_FILE_TYPES = STUDY_FILE_TYPE_NAME_HASH.keys.freeze
```